src/rgw/rgw_common.h

```diff
@@ -1210,6 +1211,7 @@ struct RGWBucketInfo
      ::encode(swift_versioning, bl);
      if (swift_versioning) {
        ::encode(swift_ver_location, bl);
+       ::encode(swift_his_location, bl);
```

Comment:
To preserve compatibility with older struct versions, we should encode new fields at the end, not in the middle. This is required during e.g. cluster upgrades when many instances of RadosGW, possibly from two different releases, may coexist in a single cluster.

Reply:
@rzarzynski seems to be fixed now?